### PR TITLE
docs(analytics): refined react-native-navigation setCurrentScreen example

### PR DESCRIPTION
**Pull request by Peeeep**
### Description

The `componentDidAppear` event is better suited than the `command` event, which doesn't fire when popping a screen natively (Android back button, iOS swipe back gesture) and in some other edge cases (e.g. `setRoot` with two components in the stack)

In addition, the componentId (usually auto-generated) is quite useless as a screen name, and componentName should be used instead.

### Checklist

- I read the [Contributor Guide](/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

**Comment by Ehesp** [comment]
Did you mean to close this?

**Comment by Peeeep** [comment]
@Ehesp Not really, but I've stupidly renamed the branch.

File: docs/analytics/screen-tracking.md [hunk 1]
```diff
@@ -37,19 +37,19 @@ documentation on the React Navigation website.
 # React Native Navigation
 
 The [`wix/react-native-navigation`](https://github.com/wix/react-native-navigation) provides 100% native platform navigation
-for React Native apps. To manually track screens, you need to setup a command listener and manually call the
-[`setCurrentScreen`](/reference/analytics#setcurrentscreen) method the Analytics library provides:
+for React Native apps. To manually track screens, you need to setup a componentDidAppear event listener and manually call the
```
Comment (Salakar) [inline docs/analytics/screen-tracking.md:40]:
```suggestion
for React Native apps. To manually track screens, you need to setup a `componentDidAppear` event listener and manually call the
```

File: .spellcheck.dict.txt [hunk 2]
```diff
@@ -27,6 +27,7 @@ Changelog
 CLI
 CocoaPods
 codebase
+componentDidAppear
```
Comment (Salakar) [inline .spellcheck.dict.txt:30]:
```suggestion
```

**Comment by Ehesp** [comment]
Thanks for this!

**Pull request merged**